lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -1149,9 +1146,9 @@ class HorizontalLineLabel extends FlLineLabel with EquatableMixin {
   ) {
     return HorizontalLineLabel(
       padding:
-          EdgeInsets.lerp(a.padding as EdgeInsets, b.padding as EdgeInsets, t),
+          EdgeInsets.lerp(a.padding as EdgeInsets, b.padding as EdgeInsets, t)!,
       style: TextStyle.lerp(a.style, b.style, t),
-      alignment: Alignment.lerp(a.alignment, b.alignment, t),
+      alignment: Alignment.lerp(a.alignment, b.alignment, t)!,
       labelResolver: b.labelResolver,
```

Comment:
Please add the direction property here in the lerp method

Reply:
I think the label direction doesn't matter for the linear interpolation. Am I wrong ?

Reply:
I think the label direction doesn't matter for the lerp method. Am I wrong ?

Reply:
For the boolean types, we use the destination value directly. So we can say
```
direction: b.direction,
```

Reply:
I am not sure about what you mean by 

> For the boolean types, we use the destination value directly

but I added the direction property in the lerp method